pmd-core/src/main/java/net/sourceforge/pmd/util/ClasspathClassLoader.java

```diff
@@ -96,4 +96,27 @@ public String toString() {
                 .append(StringUtils.join(getURLs(), ":"))
                 .append("] parent: ").append(getParent()).append(']').toString();
     }
+    
+    @Override
+    protected Class<?> loadClass(final String name, final boolean resolve) throws ClassNotFoundException {
+        // First, check if the class has already been loaded
+        Class<?> c = findLoadedClass(name);
+        if (c == null) {
+            if (c == null) {
```

Comment:
Nothing harmful, but the null check is done twice.

Reply:
Thanks! Bad copy-paste on my part